jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -150,6 +151,18 @@ def _link_image_data(self):
             error_on_fail=False,
             update_plugin=False)
 
+    def _check_if_many_data_with_wcs(self):
+        num_data_with_wcs = 0
+        for data in self.app.data_collection:
+            if hasattr(data.coords, 'pixel_to_world'):
+                if num_data_with_wcs:

```

Comment:
Shouldn't this be
```suggestion
                if num_data_with_wcs > 1:
```

Reply:
I don't think so, the increment happens below this line, so this is just saying if there was _already_ one with WCS, making this the second entry, then enable linking and stop looping. But let's definitely add a comment that clears this up, I can see how it is confusing to read 😬